plugins/catalog/src/alpha/pages.tsx

```diff
@@ -73,6 +73,9 @@ export const catalogEntityPage = PageBlueprint.makeWithOverrides({
       EntityContentBlueprint.dataRefs.filterExpression.optional(),
       EntityContentBlueprint.dataRefs.group.optional(),
     ]),
+    extraContextMenuItems: createExtensionInput([
```

Comment:
I didn't expect this one to be named "extra". I'd expect that context menu items instead be made first class citizens and that the plugin ships with a few predefined ones (that also get turned into extensions that use that exact blueprint). That way, the end user can rearrange and remove items in any way that they see fit.

Reply:
+1

Reply:
@freben @camilaibs I've gone ahead and renamed this to `contextMenuItems` and reimplemented the "Copy Entity URL" menu item as an extension, but I'm unsure how of a good way to actually use that extension which does not

A. Add a duplicate "Copy Entity URL" menu item to the menu in the new FE system. One would appear from the extension, and one would appear from the hardcoded `<MenuItem />` in `EntityContextMenu.tsx`
or
B. Remove the existing menu item from the legacy FE system. If we remove the hardcoded `<MenuItem />` then this works fine in the new system, but removes the menu item from the legacy system.

Any thoughts on how to approach this? Is there some way to detect if the NFS is in use and optionally render the hardcoded menu item if not?